src/js/letters/components/DownloadLetterLink.jsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Link } from 'react-router';
+
+import { apiRequest } from '../utils/helpers';
+
+class DownloadLetterLink extends React.Component {
+  constructor(props) {
+    super(props);
+    this.downloadLetter = this.downloadLetter.bind(this);
+  }
+
+  // This opens a duplicate window for some inexplicable reason,
```

Comment:
@annekainicUSDS This experiment was mystifying. Even without opening a window or creating the blob URL, simply issuing a POST  with an empty response opens a new window. I don't see anything in the blob() docs that explains why this would happen, eg.  https://developer.mozilla.org/en-US/docs/Web/API/Body/blob or https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch